packages/compass-settings/src/components/settings/settings-list.tsx

```diff
@@ -82,7 +88,8 @@ function SettingLabel({ name }: { name: SupportedPreferences }) {
           </span>
         )}
       </Label>
-      {long && <Description>{long}</Description>}
+      {long && !longReact && <Description>{long}</Description>}
+      {longReact && <div className={inputDescriptionStyles}>{longReact}</div>}
```

Comment:
Why does this need a completely separate component and not a Description we're already using?

```suggestion
      <Description>{longReact ?? long}</Description>
```

Reply:
I've had some problems with the component, but I've tried it again and it works good now, so I'll simplify this, thanks!